libs/pipedream.libsonnet

```diff
@@ -271,15 +271,23 @@ local pipeline_to_array(pipeline) =
     local test_pipelines = get_service_pipelines(pipedream_config, pipeline_fn, test_regions_to_render, std.length(regions_to_render) + 2);
     local rollback_pipeline = pipedream_rollback_pipeline(pipedream_config, service_pipelines, trigger_pipeline);
 
+    local all_pipelines = if parallel then pipeline_to_array(rollback_pipeline) +
+                                           pipeline_to_array(trigger_pipeline) +
+                                           // Chain the service pipelines together with
+                                           // the trigger pipeline
+                                           std.map(function(p) gocd_pipelines.chain_materials(p, trigger_pipeline), service_pipelines)
+                                           +
```

Comment:
So this works by taking each pipeline inside `service_pipelines` and chaining it to the trigger individually rather than chaining the entire `service_pipelines` to the trigger? Is that right ?

Reply:
Correct